g2p/apply.py

```diff
@@ -81,3 +88,14 @@ def run(self):
                     stdout=out,
                     stderr=err,
                 )
+
+    def filter(self):
+        fd_out, tmp_path = mkstemp(dir=".", text=True)
+        with open(self.out_g2p_lexicon, "r") as lex:
```

Comment:
```suggestion
        with uopen(self.out_g2p_lexicon, "rt") as lex:
```
to be consistent with the writing call above.?